core/storage/translation/gae_models.py

```diff
@@ -34,6 +35,153 @@
 datastore_services = models.Registry.import_datastore_services()
 
 
+class TranslatedContentDict(TypedDict):
+    """Dict type for TranslatedContent object."""
+
+    content: str|List[str]
+    needs_update: bool
+
+
+class EntityTranslationsModel(base_models.BaseModel):
+    """Model for storing entity translations ."""
+
+    entity_id = datastore_services.StringProperty(required=True, indexed=True)
+    entity_type = datastore_services.StringProperty(required=True, indexed=True)
+    entity_version = datastore_services.IntegerProperty(
+        required=True, indexed=True)
+    language_code = datastore_services.StringProperty(
+        required=True, indexed=True)
+    translations = datastore_services.JsonProperty(required=True)
```

Comment:
Should we add comments explaining each of these fields?

Reply:
Added.